demo/src/main/java/foundation/algorand/demo/OfferActivity.kt

```diff
@@ -32,6 +32,14 @@ class OfferActivity : AppCompatActivity() {
 
     private val cookieJar = Cookies()
 
+    private val iceServers = listOf(
+        PeerConnection.IceServer.builder("stun:stun.relay.metered.ca:80").createIceServer(),
```

Comment:
Same for this stun server, let's keep the google stun servers and remove the metered stun service.